doc/news/changes/minor/20191022PasqualeClaudioAfrica

```diff
@@ -0,0 +1,3 @@
+Improved: VectorTools::interpolate() now works for a parallel::distributed::Triangulation
+<br>
+(Pasquale Claudio Africa, 2019/10/22)
```

Comment:
Since I think you cooperated on this patch with @dougshidong , you might want to list both of you here :-)

Reply:
Added @dougshidong as an author.